hbase-client/src/main/java/org/apache/hadoop/hbase/client/NoncedRegionServerCallable.java

```diff
@@ -1,62 +0,0 @@
-/**
```

Comment:
We still support nonces right? In async too?

Reply:
Yes, we have nonce support in async client. All the Callables have been removed in this patch. Now we only rely on the retry logic in async client, see `AsyncRpcRetryingCallerFactory`.

Reply:
Man, so much cruft has been stripped from the client. It is great.

Reply:
It is like a startover.